File: src/com/android/recentkey/myAccessibilityService.java
package com.android.recentkey;

import android.accessibilityservice.AccessibilityService;
import android.content.Intent;
import android.os.Bundle;
import android.view.accessibility.AccessibilityEvent;

public class myAccessibilityService extends AccessibilityService {

    @Override
    public void onAccessibilityEvent(AccessibilityEvent event) {
    }

    @Override
    public void onInterrupt() {
    }

    @Override
    public int onStartCommand(Intent intent, int flags, int startId) {
        this.performGlobalAction(AccessibilityService.GLOBAL_ACTION_RECENTS);
        return super.onStartCommand(intent, flags, startId);
    }

    @Override
    public void onCreate() {
        super.onCreate();
    }
}

File: app/src/main/java/com/android/recentkey/MyAccessibilityService.kt
package com.android.recentkey

import android.accessibilityservice.AccessibilityService
import android.content.Intent
import android.view.accessibility.AccessibilityEvent

class MyAccessibilityService : AccessibilityService() {
    override fun onAccessibilityEvent(event: AccessibilityEvent) {}
    override fun onInterrupt() {}
    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
        performGlobalAction(GLOBAL_ACTION_RECENTS)
        return super.onStartCommand(intent, flags, startId)
    }

    override fun onCreate() {
        super.onCreate()
    }
}